app/networkapi/settings.py

```diff
@@ -39,7 +40,7 @@
 SECRET_KEY = env('DJANGO_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = env('DEBUG')
+DEBUG = FILEBROWSER_DEBUG = env('DEBUG')
```

Comment:
It feels like it should be possible to overwrite this value, so that if it's not defined we use whatever `DEBUG` is set to, but if it is, we use that irrespective of what `DEBUG` says?